.github/ISSUE_TEMPLATE/new_minor_release.md

```diff
@@ -0,0 +1,67 @@
+---
+name: New Minor Release
+about: Create a issue for tracking a new minor release.
+title: 'Release vX.Y.Z'
+labels: enhancement
+assignees: ''
+
+---
+
+## It is time for a new minor release for Solidus!
+
+We are committed to release every quarter in order to have a constant
+and predictable cadence for our deliverables.
+
+For more information, check out our [Release Policy](https://solidus.io/release_policy/) page.
+
+## Detailed list of steps for the release
+
+We have a pretty clear process to release a new minor version. Everything is described on our
+[Release pipeline](https://github.com/solidusio/solidus/wiki/Release-pipeline-automation) automation and [How to release Solidus](https://github.com/solidusio/solidus/wiki/How-to-release-Solidus) wiki pages,
+which we encourage to review if the assignee is not familiar with how to release Solidus.
+
+### Release preparation
+
+- Be sure there are no pending PRs that you want to include in the next minor.
+- Visit the GitHub Solidus repository.
+- Navigate to the "Actions" tab and select the "Prepare release" item.
+- Click the "Run Workflow" button, leaving `main` as the branch and the checkbox to prepare the next major unchecked (unless necesasy).
+
+At this point a new auto-generated PR will appear; be sure everything looks good and merge it.
+
+### Release the Solidus gems on Rubygems
+
+To officially release the Solidus gems, you need to have permissions on RubyGems, with 2FA enabled.
+
+Open Solidus on your local machine and run:
+
+```bash
+git fetch -a
```

Comment:
`fetch` won't update the `main` branch, right? We still need to reset the local branch or use `pull` instead.

Reply:
I think that if you are on another branch and then checkout main, you should have the latest version from the remote. But even if that's true, what I wrote doesn't cover all the scenarios, you are right, I'll just change it to 

```
git checkout main
git pull
gem release
```

Reply:
pull can generate merge commits or rebases depending on the configuration, `reset --hard` is a maybe safer alternative:

```
git checkout main
git fetch -a
git reset --hard origin/main
gem release
```

Reply:
I wanted to avoid using `origin` because not everyone (me included) has that as the main Solidus repo remote. It's so hard to update a git repo in a way that works for everyone! 🤣